pages/dr10/files.rst

```diff
@@ -523,8 +523,15 @@ match with any Legacy Surveys ``BRICK_PRIMARY`` photometric object.
 .. _`catalogs`: ../catalogs
 .. _`Catalogs`: ../catalogs
 
-survey-dr10-south-specObj-dr16.fits
------------------------------------
+survey-dr(10|10.1)-south-specObj-dr16.fits
+------------------------------------------
+
+.. note::
```

Comment:
It seems like you could just make one note that covers all of the survey files in a higher-level section.

Reply:
Thanks for this suggestion. I vacillated on whether or not to include a note with every file or just one per section, so I appreciated the second opinion.

Reply:
(and I've changed to one-note-per-file-_type_).